Source/Core/Core/IOS/ES/Identity.cpp

```diff
@@ -19,14 +19,22 @@ namespace HLE
 {
 namespace Device
 {
-IPCCommandResult ES::GetConsoleID(const IOCtlVRequest& request)
+ReturnCode ES::GetDeviceId(u32* console_id) const
```

Comment:
Is there any chance for this to ever fail? If yes, `ES::GetDeviceId(IOCtlVRequest)` should check for it, just to be safe.